org.eclipse.scout.sdk.core.s/src/main/java/org/eclipse/scout/sdk/core/s/apidef/Scout22Api.java

```diff
@@ -14,11 +14,29 @@
 
 @MaxApiLevel(22)
 @SuppressWarnings({"squid:S2176", "squid:S00118", "squid:S00100", "findbugs:NM_METHOD_NAMING_CONVENTION", "squid:S2166"}) // naming conventions
-public interface Scout22Api extends IScoutApi, IScoutChartApi, IScout22DoApi {
+public interface Scout22Api extends IScoutApi, IScout22Api, IScoutChartApi, IScout22DoApi {
 
   @Override
   default int[] supportedJavaVersions() {
     return new int[]{11, 17};
   }
 
+  IScout22Api.IQueryBuilder I_QUERY_BUILDER = new Scout22Api.IQueryBuilder();
+
+  @Override
+  default IScout22Api.IQueryBuilder IQueryBuilder() {
+    return I_QUERY_BUILDER;
+  }
+
+  class IQueryBuilder implements IScout22Api.IQueryBuilder {
+    @Override
+    public String fqn() {
+      return "com.bsiag.scout.rt.datamodel.persistence.common.IQueryBuilder";
```

Comment:
com.bsiag.scout things do not belong here as it is not available here. Move to other plugin. E.g. execute ScoutApiTest which tells you the same :)